gnoi/reset/server_test.go

```diff
@@ -60,8 +72,9 @@ func makeTests() []test {
 
 func TestStart(t *testing.T) {
 	tests := makeTests()
+	mock := &certServerMock{}
```

Comment:
Use a more descriptive name here. You could end up with lots of different things mocked and it would make it easier to understand which is which.